radis/labels/utils/processor_utils.py

```
@@ -0,0 +1,55 @@
+    fields.
+    """
+
+    choice_values = tuple(
+        choice.value
+        for choice in question.choices.all()
+        if isinstance(choice.value, str) and choice.value
+    )
+    if not choice_values:
+        raise ValueError(
+            f"LabelQuestion {question.pk or '<unsaved>'} has no valid choices configured."
+        )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)